src/metal.js

```diff
@@ -1,3 +1,23 @@
-// Add Backbone.Metal extensions here
-// 1. Marionette.Class
-// 2. Marionette.Error and Marionette.Deprecate
+// Missing Metal Items
+// 1. Marionette.Class parity
+// 2. Marionette.Deprecate
+
+Marionette.Class = Metal.Class.extend({});
+
+function classify(obj) {
+  return Marionette.Class.extend(
+    _.extend(
+      {constructor: obj},
+      _.omit(obj.prototype, _.keys(Marionette.Class.prototype))
+    )
```

Comment:
Is this line necessary?  https://github.com/thejameskyle/backbone-metal-classify/blob/master/index.js#L13